courses/free-courses-hi.md

```diff
@@ -104,6 +104,7 @@
 ### Bash and Shell
 
 * [Bash Basic Commands](https://youtube.com/playlist?list=PLzOLSdbK1deOKmOiiv-o4wn7xUj6ZYzrM) - Noob Coders
+* [Bash scripting tutorial for beginners | learn shell scripting in one video in Hindi English](https://www.youtube.com/watch?v=9T2nEXlLy9o&list=PLxLRoXCDIalcosmDOQizh31EIHEK1njfO&index=1&ab_channel=FortifySolutions) - Fortify Solutions (Savita Dadhwal, Soniya)
```

Comment:
Please change this link to playlist link

Reply:
Done, Thanks.